### PR TITLE
parquet: updated parquet-go dependency to v1.6.2 for not needing CGO_…

### PR DESCRIPTION
**Pull request by applike-ss**
…ENABLED=1 anymore;

### CODE REVIEW

**Comment by applike-ss** [comment]
@selm0 since you are the parquet master, it would be great if you could revisit if there are any problematic changes in the library update.

File: go.mod [hunk 1]
```diff
@@ -76,91 +76,12 @@ require (
 )
 
 require (
-	cloud.google.com/go v0.54.0 // indirect
 	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
-	github.com/DataDog/zstd v1.4.8 // indirect
```
Comment (selm0) [inline go.mod:81]: are all of these dependenices now obsolete? :thinking:

**Review by selm0 — APPROVED** [review]

**Comment by nikplxjt** [comment]
I tried it out with a minimal example (using https://gitlab.justdice-ops.io/paulicks/spend-event-to-parquet) to write a simple parquet file in local S3. The S3 writing process returns with the following error: 
```
error during the execution of stage 2048: panic during running of module converter: failed to create schema from tag map: type UTF8: not a valid Type string  application
```

The used schema looks like this:
```json
{"Tag":"name=parquet-go-root, repetitionType=REQUIRED","Fields":[{"Tag":"repetitionType=REQUIRED, name=id, type=UTF8"},{"Tag":"type=UTF8, repetitionType=REQUIRED, name=user_uuid"},{"Tag":"name=user_install_id, type=UTF8, repetitionType=REQUIRED"},{"Tag":"repetitionType=REQUIRED, name=campaign_id, type=UINT_32"},{"Tag":"type=UINT_32, repetitionType=REQUIRED, name=network_id"},{"Tag":"type=UINT_32, repetitionType=REQUIRED, name=channel_id"},{"Tag":"type=UINT_32, repetitionType=REQUIRED, name=country_id"},{"Tag":"name=source_id, type=UTF8, repetitionType=REQUIRED"},{"Tag":"name=source_bundle_id, type=UTF8, repetitionType=REQUIRED"},{"Tag":"name=adset_name, type=UTF8, repetitionType=REQUIRED"},{"Tag":"name=external_ad_set_id, type=UTF8, repetitionType=REQUIRED"},{"Tag":"name=spend_value, type=FLOAT, repetitionType=REQUIRED"},{"Tag":"repetitionType=REQUIRED, name=spend_value_eur, type=FLOAT"},{"Tag":"type=FLOAT, repetitionType=REQUIRED, name=spend_value_usd"},{"Tag":"type=UTF8, repetitionType=REQUIRED, name=spend_currency"},{"Tag":"type=UTF8, repetitionType=REQUIRED, name=source_name"},{"Tag":"name=exchange_id, type=UTF8, repetitionType=REQUIRED"},{"Tag":"name=exchange_name, type=UTF8, repetitionType=REQUIRED"},{"Tag":"name=created_at, type=INT96, repetitionType=REQUIRED"},{"Tag":"name=happened_at, type=INT96, repetitionType=REQUIRED"}]}
```

I think this is due to some change in parquet-go since which one has to use the primitive type in the `type` tag and add the converted type in the `convertedType` type. See https://github.com/xitongsys/parquet-go/blob/master/README.md for details. When I used the "raw" parquet-go on the latest version I also had to do it in the way that is described in the readme in order to get it working. In the `parquet` package we currently only use the `type` tag.

File: go.mod (hunk 1 above)
Comment (applike-ss) [reply go.mod:81]: i often do `go mod tidy`, so yes :-)

**Comment by applike-ss** [comment]
I will check and fix that. Let me document that here.

With the old implementation i get this schema:
```json
{
  "Tag": "name=parquet-go-root, repetitionType=REQUIRED",
  "Fields": [
    {
      "Tag": "name=id, type=UINT_32, repetitionType=REQUIRED"
    },
    {
      "Tag": "repetitionType=REQUIRED, name=uuid, type=UTF8"
    },
    {
      "Tag": "repetitionType=REQUIRED, name=value, type=FLOAT"
    },
    {
      "Tag": "type=INT96, repetitionType=REQUIRED, name=created_at"
    }
  ]
}
```
With the new implementation i get this schema:
```json
{
  "Tag": "name=parquet-go-root, repetitionType=REQUIRED",
  "Fields": [
    {
      "Tag": "name=id, type=UINT_32, repetitionType=REQUIRED"
    },
    {
      "Tag": "name=uuid, type=UTF8, repetitionType=REQUIRED"
    },
    {
      "Tag": "name=value, type=FLOAT, repetitionType=REQUIRED"
    },
    {
      "Tag": "type=INT96, repetitionType=REQUIRED, name=created_at"
    }
  ]
}
```
So the schema is still the same, but the behaviour of the library changed.

**Pull request closed without merging**